File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/WishList/Sort.java

package com.artmurka.artmurkaapp.Model.Pojo.ItemList.WishList;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Sort {

    @SerializedName("sort")
    @Expose
    private String sort;
    @SerializedName("order")
    @Expose
    private String order;
    @SerializedName("slist")
    @Expose
    private List<Slist> slist = null;

    public String getSort() {
        return sort;
    }

    public void setSort(String sort) {
        this.sort = sort;
    }

    public String getOrder() {
        return order;
    }

    public void setOrder(String order) {
        this.order = order;
    }

    public List<Slist> getSlist() {
        return slist;
    }

    public void setSlist(List<Slist> slist) {
        this.slist = slist;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/wishList/Sort.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.wishList

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Sort {

    @SerializedName("sort")
    @Expose
    var sort: String? = null
    @SerializedName("order")
    @Expose
    var order: String? = null
    @SerializedName("slist")
    @Expose
    var slist: List<Slist>? = null

}
